src/values.rs

```diff
@@ -109,9 +133,50 @@ impl Ord for OnOff {
     }
 }
 
-/// An open/closed state. Internal representation may be either
-/// Open or Closed.
-#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+impl Serialize for OnOff {
+    fn serialize<S>(&self, serializer: &mut S) -> Result<(), S::Error> where S: Serializer {
+        match *self {
+            OnOff::On => "On".serialize(serializer),
+            OnOff::Off => "Off".serialize(serializer)
+        }
+    }
+}
+impl Deserialize for OnOff {
+    fn deserialize<D>(deserializer: &mut D) -> Result<Self, D::Error> where D: Deserializer {
+        deserializer.deserialize_string(TrivialEnumVisitor::new(|source| {
+            match source {
+                "On" | "on" | "ON" => Ok(OnOff::On),
```

Comment:
I don't think that there will be any meaningful impact, but I realize that we probably shouldn't do that just for some deserializers and not for others. So let's roll back to keeping only the documented format, for the time being, and then, if we find a way to make it work for all enumerations, consider generalizing to case variations?

Does this sound good to you?